tools/db_bench_tool.cc

```diff
@@ -3307,18 +3318,35 @@ class Benchmark {
                 max_num_range_tombstones_ &&
             num_written % writes_per_range_tombstone_ == 0) {
           int64_t begin_num = key_gens[id]->Next();
-          GenerateKeyFromInt(begin_num, FLAGS_num, &begin_key);
-          GenerateKeyFromInt(begin_num + range_tombstone_width_, FLAGS_num,
-                             &end_key);
-          if (FLAGS_use_blob_db) {
-            s = db_with_cfh->db->DeleteRange(
-                write_options_, db_with_cfh->db->DefaultColumnFamily(),
-                begin_key, end_key);
-          } else if (FLAGS_num_column_families <= 1) {
-            batch.DeleteRange(begin_key, end_key);
+          if (FLAGS_expand_range_tombstones) {
+            for (int64_t offset = 0; offset < range_tombstone_width_;
+                 ++offset) {
+              GenerateKeyFromInt(begin_num + offset, FLAGS_num,
+                                 &expanded_keys[offset]);
+              if (FLAGS_use_blob_db) {
+                s = db_with_cfh->db->Delete(write_options_,
```

Comment:
Oh, it's optional. Our very old DB functions like Delete() have overloads that don't require pointer-to-ColumnFamilyHandle and will use the one for default CF. Those are left around for compatibility reasons. But for newer DB functions, like DeleteRange(), we stopped providing such overloads so now the user always needs to provide pointer-to-ColumnFamilyHandle explicitly.